network-store-iidm-tck/src/test/java/com/powsybl/network/store/tck/GroundIT.java

```diff
@@ -30,7 +30,7 @@
 class GroundIT extends AbstractGroundTest {
 
     @Test
-    void testOnSubnetwork() {
+    public void testOnSubnetwork() {
```

Comment:
It's a `org.junit.jupiter.api.Test` so JUnit 5 test, so not `public` visibility.

Also remove `@RunWith(SpringRunner.class)` which is JUnit 4, it don't mix with JU5.

Reply:
We can't remove public here, because method overrides public method in abstract base class in powsybl-core ...

All other existing tck test classes have already this annotation : @RunWith(SpringRunner.class) : removing it is not part of the upgrade scope ...

Reply:
oh yes, I haven't see the method was in parent.  
In that case the override tag is missing.
```suggestion
    @Override
    public void testOnSubnetwork() {
```

After there is a better solution with `@Disabled` to not have a green test masking the truth:
```java
    @Test
    @org.junit.jupiter.api.Disabled("Subnetworks are not implemented")
    @Override
    public void testOnSubnetwork() {
        // FIXME remove when subnetworks are implemented
    }
```

Reply:
> oh yes, I haven't see the method was in parent. In that case the override tag is missing.
> 
> After there is a better solution with `@Disabled` to not have a green test masking the truth:
> 
> ```java
>     @Test
>     @org.junit.jupiter.api.Disabled("Subnetworks are not implemented")
>     @Override
>     public void testOnSubnetwork() {
>         // FIXME remove when subnetworks are implemented
>     }
> ```

I agree we could do that @Tristan-WorkGH but since it's done that way already in all the tck (and we want to be homogeneous) let's not change the 30+ tck files in this PR that isn't about that.